### PR TITLE
Display json to page dynamically

### PR DESCRIPTION
**Pull request by aseebart**

### CODE REVIEW

**Comment by aseebart** [comment]
I was able to build the list of plays dynamically. Does anyone object? If not please merge or I will later today.

File: statler_site/templates/play-list-page.html [hunk 1]
```diff
@@ -14,16 +14,23 @@
       $(document).ready(function(){
         // Get JSON from a URL as an object
         $.getJSON(apiEndpointUrl, function(result){
-          // display the returned json
-          $('p').append(JSON.stringify(result));
+          // display the returned json dynamically
+          $.each(result, function(key, value){
+            $("<li/>").appendTo("#plays").html("<h3>" + value.fields.title + "</h3");
```
Comment (bencdelaney) [inline statler_site/templates/play-list-page.html:19]: Is that closing h3 tab missing a bracket?

File: statler_site/templates/play-list-page.html [hunk 2]
```diff
@@ -14,16 +14,23 @@
       $(document).ready(function(){
         // Get JSON from a URL as an object
         $.getJSON(apiEndpointUrl, function(result){
-          // display the returned json
-          $('p').append(JSON.stringify(result));
+          // display the returned json dynamically
+          $.each(result, function(key, value){
```
Comment (bencdelaney) [inline statler_site/templates/play-list-page.html:18]: The first argument is more of an index than a key, correct? (This is the sort of nitpicky note you're welcome to ignore if you disagree.)

File: statler_site/templates/play-list-page.html [hunk 3]
```diff
@@ -14,16 +14,23 @@
...
+            $("<li/>").appendTo("#plays").html("<h3>" + value.fields.title + "</h3");
+            $("<p/>").appendTo("#plays").html(value.fields.title);
+            $("<p/>").appendTo("#plays").html(value.fields.director);
```
Comment (bencdelaney) [inline statler_site/templates/play-list-page.html:21]: It's sub-optimal that we need to dot-into the "fields" object. We should fix this on the API side. (Relevant to #35, #38, #39)

**Comment by bencdelaney** [comment]
Looks good, Anthony! Merge it at will.

**Comment by mnijmeh11** [comment]
Cool...

-----Original Message-----

**Pull request merged**